source/intro.tex

```diff
@@ -1,6 +1,15 @@
 %!TEX root = std.tex
 
+\clearpage
+\bigskip\noindent\textlarger{\textbf{%
+Information Technology —\\[1.5ex]Programming languages, their environments and system software interfaces —\\[1.5ex]\Cpp{}}}
+\bigskip\bigskip
+
+\begingroup
+\let\clearpage\relax
```

Comment:
@Eelis I hope this won't be too annoying. It's not so easy to prevent the page break before a chapter start, so this quick hack gets the job done.

Reply:
No worries, I'll manage. Cheers!